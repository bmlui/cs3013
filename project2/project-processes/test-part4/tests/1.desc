Testing background jobs, 1 job 
