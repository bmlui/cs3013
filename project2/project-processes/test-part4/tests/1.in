sleep 1 &
sleep 2
jobs
exit

