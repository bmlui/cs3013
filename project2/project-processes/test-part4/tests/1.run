./wshell < tests/1.in
