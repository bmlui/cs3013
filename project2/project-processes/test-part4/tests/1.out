test-part4$ sleep 1 &
[1]
test-part4$ sleep 2
[1] Done: sleep 1
test-part4$ jobs
test-part4$ exit
